lib/cli.ex

```diff
@@ -4,11 +4,15 @@ defmodule Onigumo.CLI do
   }
 
   def main(argv) do
-    case OptionParser.parse(argv, strict: []) do
-      {[], [component], []} ->
+    case OptionParser.parse(
+           argv,
+           aliases: [C: :working_dir],
+           strict: [working_dir: :string]
+         ) do
+      {parsed_switches, [component], []} ->
         {:ok, module} = Map.fetch(@components, String.to_atom(component))
-        root_path = File.cwd!()
-        module.main(root_path)
+        working_dir = Keyword.get(parsed_switches, :working_dir, File.cwd!())
```

Comment:
_root_path_ has been renamed to working_dir to correspond with the switch name, right? _working_dir_ is definitely a better name, _root_path_ may be confusing.

Reply:
🤙

Reply:
I found my issue #222 where I suggested the rename. Linked.